### PR TITLE
fix: can create team within existing org

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/8511

### To test

- [ ] Create a new team within an existing org without the page crashing

### CODE REVIEW

File: packages/client/modules/newTeam/components/NewTeamForm/NewTeamForm.tsx [hunk 1]
```diff
@@ -129,7 +129,7 @@ const NewTeamForm = (props: Props) => {
   const disableFields = !!lockedSelectedOrg && !isNewOrg
   const selectedOrg = organizations.find((org) => org.id === orgId)
   const selectedOrgTeamMemberEmails = selectedOrg?.teams.flatMap(({teamMembers}) =>
-    teamMembers.filter(({isSelf}) => !isSelf).map(({email}) => email)
+    teamMembers?.filter(({isSelf}) => !isSelf).map(({email}) => email)
```
Comment (igorlesnenko) [inline packages/client/modules/newTeam/components/NewTeamForm/NewTeamForm.tsx:132]: Not sure I understand, in what cases `teamMembers` will be null? I see we are fetching them above in graphql query

Comment (nickoferrall) [reply packages/client/modules/newTeam/components/NewTeamForm/NewTeamForm.tsx]: Looks like the `teamMembers` haven't populated in the newly created team. I've updated the `AddTeamMutation` to query for the `teamMembers` as that's a cleaner way of doing it

File: packages/client/mutations/AddTeamMutation.ts [hunk 2]
```diff
@@ -19,6 +19,9 @@ graphql`
     team {
       id
       name
+      teamMembers {
```
Comment (igorlesnenko) [inline packages/client/mutations/AddTeamMutation.ts:22]: Can we spread a fragment here? teamMembers are not used in this file directly, so it is unknown why are they here

**Review by igorlesnenko — APPROVED** [review]

File: packages/client/mutations/AddTeamMutation.ts (hunk 2 above)
Comment (nickoferrall) [reply packages/client/mutations/AddTeamMutation.ts]: I like it! Updated 👍

**Pull request merged**